### PR TITLE
Corrections & Additions to Tsu SSB.

### PR DESCRIPTION
**Pull request by RagiingAlways**

### CODE REVIEW

File: mods/tsunamissb/statuses.js [hunk 1]
```diff
@@ -97,10 +97,10 @@ exports.BattleStatuses = {
 	tsunamiprince: {
 		exists: true,
 		onStart: function () {
-			this.add('c', '~Tsunami Prince', 'You think i came for the battle, when it\'s really your soul i want.');
+			this.add('c', '~Tsunami Prince', 'You think I came for the battle, when it\'s really your soul i want.');
```
Comment (DeathlyPlays) [inline mods/tsunamissb/statuses.js:100]: Capitalize the "I"?

**Review by DeathlyPlays — COMMENTED** [review]
Nitpicky meh

File: mods/tsunamissb/statuses.js (hunk 1 above)
Comment (RagiingAlways) [reply mods/tsunamissb/statuses.js:100]: damn you insist I missed one
Now I'll have to fix that next update.

Comment (DeathlyPlays) [reply mods/tsunamissb/statuses.js:100]: Rekt

Comment (RagiingAlways) [reply mods/tsunamissb/statuses.js:100]: By the by, ever noticed that one with HH? XD that's on SG too. ~~ik you have touched their file more than I~~

Comment (RagiingAlways) [reply mods/tsunamissb/statuses.js:100]: https://github.com/DesoGit/Tsunami_PS/pull/21
https://github.com/HoeenCoder/SpacialGaze/pull/312
 :P there corrected both in both places as I was already doing another PR for each today anyways. 
Kraken Mare, here to correct the world one word at a time. @DeathlyPlays

Comment (DeathlyPlays) [reply mods/tsunamissb/statuses.js:100]: Lol

**Pull request merged**